acceptance/clients/conditions_test.go

```diff
@@ -0,0 +1,66 @@
+//go:build acceptance
+// +build acceptance
```

Comment:
Please put that file into a `testing directory` or this will be skipped by the unittest (in fact coverage) job. See https://github.com/gophercloud/gophercloud/blob/4f6d1451acac9260a6901451e146b2bee4742bb5/script/coverage#L6